PUBLISH.md

```diff
@@ -2,13 +2,18 @@
 
 The module is published to [RubyGems][1] under the gem name [plaid][2].
 
-To publish:
+Prepare release:
 
-1. `rake test` (runs the test suite locally).
-2. Update, commit, and merge (with review) the `lib/plaid/version.rb` and `CHANGELOG.md` files. 
-5. `git pull` (makes sure your `HEAD` is up-to-date).
-6. `rake release` (builds the gem, creates a tag, pushes the gem to RubyGems and tag to GitHub).
-7. `rake update_github_docs` (generates RDoc files, updates `gh-pages` branch and pushes it to GitHub). (If you run into a `gh-pages` branch error, you may need to manually run `git checkout gh-pages` to ensure the branch is checked out locally.)
+1. update `lib/plaid/version.rb`, and `CHANGELOG.md` files.
+2. run `bundle` to bump the version in `Gemfile.lock`
+3. create and merge a PR with the changes from 1 and 2.
+
+Publish:
+
+1. `git checkout master` and `git pull` (makes sure your `HEAD` is up-to-date).
+2. Check that tests are passing on latest `master` build and (optional) `bundle exec rake test` to run tests locally
```

Comment:
hmm `bundle exec` doesn't work for me but running `rake test` does @skylarmb

Reply:
nvm, figured it out. It was because I had was using a weird version of bundler. This works for me now

Reply:
everything should be run using bundler otherwise you are using your system-wide / globally installed gems.